common/src/main/java/com/wynntils/screens/overlays/placement/OverlayManagementScreen.java

```diff
@@ -239,12 +242,7 @@ public void doRender(PoseStack poseStack, int mouseX, int mouseY, float partialT
             }
         }
 
-        super.doRender(poseStack, mouseX, mouseY, partialTick); // This renders widgets
-        // This renders button tooltips
-        if (this.deferredTooltipRendering != null) {
```

Comment:
This is not needed anymore? Is deferredTooltipRendering a dead variable then?

Reply:
This is called by MC now, after render call. The same variable works just like it (in fact we have technical debt of not using it).